index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+thebroadwalk.com
+        </title>
+  <link rel="stylesheet" href="eventhall.css">
+    </head>
+    <body>
+<header></header> <img src="header.png"> alt="header image" </header>
+<div class="topnav">
+ <a class="active" href="#Home">Home</a>
+    <a href="contact">Contact</a>
```

Comment:
clicking on the contact menu should take you to the contact page. Do the same for others.